src/__tests__/reader.test.js

```diff
@@ -0,0 +1,23 @@
+import { readFileSync } from 'fs';
+import { join } from 'path';
+
+import { readBRML } from '../reader/reader';
+
+describe('readBRML', () => {
+  const data = readFileSync(join(__dirname, '../../data/test.brml'));
+
+  it('check the output dictionary', async () => {
+    let result = await readBRML(data);
```

Comment:
Please also check some values. At worse you can use https://jestjs.io/docs/en/snapshot-testing

Reply:
Tests on values added in https://github.com/cheminfo/xrd/pull/3/commits/da0378de81d8d6650c1fed400834b62e3722d7f7.

Reply:
Will also move them to the refactored version (https://github.com/kjappelbaum/xrd/tree/spectrum_class).